### ISSUE TITLE
it should be much more obvious that errors can be clicked on

### ISSUE THREAD

**Issue by dmd** [poldracklab/fmriprep#1991]
![image](https://user-images.githubusercontent.com/41439/74941370-e4763600-53c0-11ea-83f8-49e47ebcdd88.png)

It is not at _all_ obvious that these are expandable by clicking on them. Instead of round bullets, these should be rotating triangles.

**Comment by effigies** [comment]
@mgxd @oesteban I'd suggest that aesthetic fixes in reports are safe to put into bug-fix releases, as they don't change any APIs or workflow structures. WDYT?

**Comment by mgxd** [comment]
I agree

**Comment by effigies** [comment]
This can probably be very easily changed by replacing the current [template](https://github.com/poldracklab/niworkflows/blob/0aa6ddea470bf7228ee649af71a08a86e0ec5ff7/niworkflows/reports/report.tpl#L133-L146):

```Jinja
        <div class="nipype_error">
            Node Name: <a target="_self" onclick="toggle('{{error.file|replace('.', '')}}_details_id');">{{ error.node }}</a><br>
            <div id="{{error.file|replace('.', '')}}_details_id" style="display:none">
            File: <code>{{ error.file }}</code><br>
            Working Directory: <code>{{ error.node_dir }}</code><br>
            Inputs: <br>
            <ul>
            {% for name, spec in error.inputs %}
                <li>{{ name }}: <code>{{ spec }}</code></li>
            {% endfor %}
            </ul>
            <pre>{{ error.traceback }}</pre>
            </div>
        </div>
```

With something like:

```Jinja
        <details class="nipype_error">
            <summary>Node Name: {{ error.node }}</summary><br>
            <div id="{{error.file|replace('.', '')}}_details_id" style="display:none">
            File: <code>{{ error.file }}</code><br>
            Working Directory: <code>{{ error.node_dir }}</code><br>
            Inputs: <br>
            <ul>
            {% for name, spec in error.inputs %}
                <li>{{ name }}: <code>{{ spec }}</code></li>
            {% endfor %}
            </ul>
            <pre>{{ error.traceback }}</pre>
            </div>
        </details>
```

**Comment by effigies** [comment]
Closed by 2590fed406a77874f6ab5dc498d246bc0f0db4bc.
